src/answers-umd.js

```diff
@@ -388,6 +401,24 @@ class Answers {
     this.core.globalStorage.set('queryTrigger', 'initialize');
     this.core.setQuery(searchConfig.defaultInitialSearch);
   }
+
+  /*
+   * Updates the css styles with new current variables. This is useful when the css
+   * variables are updated dynamically (e.g. through js) or if the css variables are
+   * added after the ANSWERS.init
+   * @param {Object} config Additional config to pass to the ponyfill
+   */
+  ponyfillCssVariables (config = {}) {
```

Comment:
I think, for now, it's sufficient to just give people hooks into onFinally, onSuccess, and onError. The other callbacks seem more specialized. We can give people access to these if they request it later on.

Reply:
you bet